src/main/java/edu/kis/vh/nursery/stack/IntArrayStack.java

```diff
@@ -0,0 +1,46 @@
+package edu.kis.vh.nursery.stack;
+
+public class IntArrayStack implements IntStack {
+	public final static int MAX_NUMBERS_COUNT = 12;
+
+	private final int[] numbers = new int[MAX_NUMBERS_COUNT];
+
+	private int total = STACK_EMPTY;
+
+	public int getTotal() {
+		return total;
+	}
+
+	@Override
+	public void push(int in) {
+		if (!isFull()) {
+			numbers[total++] = in;
```

Comment:
Changing STACK_EMPTY value requires such fixes? It does not make sense. Incorrect fix.